test/integration/stock/lots.js

```diff
@@ -0,0 +1,46 @@
+/* global expect, agent */
+/* jshint expr : true */
+
+const moment = require('moment');
+const helpers = require('../helpers');
+const shared = require('./shared');
+
+describe('(/lots/) The lots HTTP API', () => {
+  it('GET /lots/:uuid returns details of a lot', () => {
+    return agent.get(`/lots/${shared.lotQuinineUuid}`)
+      .then((res) => {
+        expect(res).to.have.status(200);
+        const expectedKeys = [
+          'uuid', 'label', 'quantity', 'unit_cost', 'description',
+          'expiration_date', 'inventory_uuid', 'text',
+        ];
+        expect(res.body).to.have.all.keys(expectedKeys);
+      })
+      .catch(helpers.handler);
+  });
+
+  it('PUT /lots/:uuid update lot label or expiration date', () => {
+    const update = {
+      label : 'Lot Quinine Updated',
+      expiration_date : '2020-12-15',
+      unit_cost : 1.7,
+    };
+
+    return agent.put(`/lots/${shared.lotQuinineUuid}`)
+      .send(update)
+      .then((res) => {
+        expect(res).to.have.status(200);
+        return agent.get(`/lots/${shared.lotQuinineUuid}`);
+      })
+      .then(res => {
+        const details = res.body;
+        expect(details.label).to.be.equal(update.label);
+        expect(formatDate(details.expiration_date)).to.be.equal(update.expiration_date);
```

Comment:
As an alternative to using `formatDate()`, I'm pretty sure this will work:
```js
const update = {
       expiration_date : new Date('2020-12-15'),
};

// later ....
      .then(res => {
        const details = res.body;
        expect(details.label).to.be.equal(update.label);
        expect(details.expiration_date).to.be.equal(update.expiration_date);
      });
```

Could you try it and see if it works?

Reply:
The server return the date in the string format, here is the result of the test : 
![image](https://user-images.githubusercontent.com/5445251/49927035-5863a700-febd-11e8-9dbd-7b5d804598e5.png)

Reply:
Alright, thank you for trying! :+1: